### DISCUSSION TITLE
next, previous, back, open file

### DISCUSSION THREAD

**Discussion by mitobin** [rejetto/hfs#248] [Q&A] [answered]
Hello Rejetto!
I find Seagate has a very nice feature: next, previous, back, open files. this feature is needed when manipulating files, can you put it on hfs3?

https://github.com/rejetto/hfs/assets/132363335/7f952b54-e1a4-4973-ac1a-b2db3a4908e2

**Comment by rejetto** [comment]
It's nice, yes, but at the moment HFS has no "visualization" feature at all. 🙃
I'm leaving visualization to plugins, as I don't consider it "core".
I'll do my best to make it possible for plugins to offer such feature.

**Reply by mitobin** [reply]
Thank you so much for your dedication 🙏🙏🙏

**Comment by mitobin** [comment, marked as answer]
I found the answer at #270. it works better than i expected. Thanks 🙏🙏🙏
